code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/notification/templates/InputBoxPushTemplate.kt

```diff
@@ -0,0 +1,61 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.services.ui.notification.templates
+
+import android.content.Intent
+import com.adobe.marketing.mobile.services.ui.notification.PushTemplateConstants
+import com.adobe.marketing.mobile.util.DataReader
+
+internal class InputBoxPushTemplate : AEPPushTemplate {
+    // Required, the intent action name to be used when the user submits the feedback.
```

Comment:
Since this is required - can you make it `lateinit` and non optional.
You will also have to throw an exception when this filed does not exist.